collect_app/src/main/res/drawable/baseline_check_24.xml

```diff
@@ -0,0 +1,5 @@
+<vector android:height="24dp" android:tint="#000000"
```

Comment:
In other icons if it's black we use `android:tint="@android:color/black"` so let's do the same here.